js/src/atk-utils.js

```diff
@@ -37,4 +37,24 @@ const atkEventBus = (function () {
     };
 }());
 
-export { atkOptions, atkEventBus };
+/*
+* Utilities function that you can execute
+* from atk context. Usage: atk.utils.date().parse('string');
+*/
+const atkUtilites = (function () {
```

Comment:
atkUtils to be consistent with `atk.utils.`

Reply:
done!